Assets/MRTK/SDK/Editor/Inspectors/UX/Interactable/ButtonConfigHelperInspector.cs

```diff
@@ -160,20 +164,55 @@ public override void OnInspectorGUI()
+
                                 if (showComponents)
                                 {
                                     EditorGUILayout.PropertyField(seeItSayItLabelTextProp);
                                 }
 
-                                EditorGUI.BeginChangeCheck();
+                                showSpeechCommand = EditorGUILayout.Toggle("Display Speech Command", showSpeechCommand);
 
                                 SerializedObject sisiLabelTextObject = new SerializedObject(seeItSayItLabelTextProp.objectReferenceValue);
                                 SerializedProperty sisiTextProp = sisiLabelTextObject.FindProperty("m_text");
-                                EditorGUILayout.PropertyField(sisiTextProp, new GUIContent("See it / Say it Label"));
-                                EditorGUILayout.Space();
+                                if (!showSpeechCommand)
+                                {
+                                    EditorGUILayout.PropertyField(sisiTextProp, new GUIContent("See it / Say it Label"));
+                                    EditorGUILayout.Space();
+                                }
+                                else
+                                {
+                                    if (interactableProp.objectReferenceValue != null)
+                                    {
+                                        Debug.Log("hitting here");
+                                        SerializedObject interactableObject = new SerializedObject(interactableProp.objectReferenceValue);
+                                        SerializedProperty voiceCommandProperty = interactableObject.FindProperty("VoiceCommand");
+
+                                        if(string.IsNullOrEmpty(voiceCommandProperty.stringValue))
+                                        {
+                                            EditorGUILayout.HelpBox("No valid speech command provided to the interactable", MessageType.Warning);
+                                        }
+                                        else
+                                        {
+                                            string sisiText = string.Format("Say \"{0}\"", voiceCommandProperty.stringValue);
+                                            if (sisiTextProp.stringValue != sisiText)
+                                            {
+                                                sisiTextProp.stringValue = sisiText;
+                                                sisiChanged = true;
+                                            }
+                                        }
+                                    }
+                                    else
+                                    {
+                                        EditorGUILayout.HelpBox("There is no interactable attached linked to the button config helper. One is needed to display the appropriate speech command", MessageType.Warning);
```

Comment:
I'm not sure the right wording here, but for whatever reason "There is no interactable attached linked to the button config helper." reads weirdly to me. I think it's the "interactable attached linked" part. Should one of "attached" or "linked" be removed? Or am I reading it entirely wrong?

Reply:
Yeah it's worded poorly, I think it should be "There is not interactable linked to the button config helper. One is needed..."